config-connector/solutions/iam/kpt/member-iam/README.md

```diff
@@ -0,0 +1,84 @@
+Member IAM
+==================================================
+
+# NAME
+
+  member-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (defaults to `compute.networkAdmin`) in the project.
+
+  You can also create any number of service accounts and grant them any number of roles by altering and applying the altered config YAMLs.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account 
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following 
+  roles in your desired project (it doesn't need to be the project where you 
+  installed Config Connector):
+
+  - roles/resourcemanager.projectIamAdmin
+  - roles/iam.serviceAccountAdmin
+
+# USAGE
+
+  Replace `${PROJECT_ID?}` with your desired project ID value from 
+  within this directory:
+
+  ```
+  kpt cfg set . project-id VALUE
+  ```
+
+  Once the project ID is set in the configs, simply apply the YAMLs:
+
+  ```
+  kubectl apply -f .
+  ```
+
+## OPTIONAL WORKFLOW
```

Comment:
I feel like we may want to hold off on an optional workflow section, or make it more targeted instead of just appending it to # USAGE... I feel that as is it doesn't help too much, and the **Grant Multiple Roles** section especially seems unclear to me.

Reply:
As we discussed offline, let's just make it another top level section in README.